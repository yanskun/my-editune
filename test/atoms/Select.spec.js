import { shallowMount } from '@vue/test-utils'
import Component from '~/components/atoms/select'

describe('Select', () => {
  test('snapshot', () => {
    const wrapper = shallowMount(Component, {
      props: {},
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
