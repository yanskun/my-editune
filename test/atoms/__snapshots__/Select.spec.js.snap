// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select snapshot 1`] = `
<el-select-stub
  autocomplete="off"
  multiplelimit="0"
  popperappendtobody="true"
  valuekey="value"
/>
`;
